runtime/interpreter/interpreter.go

```diff
@@ -1085,10 +1085,15 @@ func (interpreter *Interpreter) InvokeExternally(
 		preparedArguments[i] = interpreter.ConvertAndBox(getLocationRange, argument, nil, parameterType)
 	}
 
+	var self *CompositeValue
+	if boundFunc, ok := functionValue.(BoundFunctionValue); ok {
+		self = boundFunc.Self
+	}
```

Comment:
Does this have any effect on anything else outside of the testing framework? Seems like changing the receiver of an externally invoked function could have other consequences.

Reply:
It shouldn't really affect others. `InvokeExternally` is used to invoke a cadence function from Go. For bound functions (e.g: `foo.bar()`), the receiver information was missing. Perhaps it was never needed before.